nengo_loihi/neurons.py

```diff
@@ -321,35 +321,41 @@ class NoiseBuilder(object):
 
         Attributes
         ----------
-        models : list of NeuronOutputNoise
+        noise_models : list of NeuronOutputNoise
             The noise models used for each op/signal.
         """
+
         builders = {}
 
-        def __init__(self, ops, signals, config, models):
-            self.models = models
+        def __init__(self, ops, signals, config, noise_models):
+            self.noise_models = noise_models
             self.dtype = signals.dtype
             self.np_dtype = self.dtype.as_numpy_dtype()
-            self.np_ones = [
-                np.ones((op.J.shape[0], 1), dtype=self.np_dtype) for op in ops]
+
+            if hasattr(noise_models[0], "tau_s"):
```

Comment:
I don't like this. I always bristle at `hasattr`/`getattr`, even though they do have their uses sometimes. Avoiding this is why I created the RCNoiseModel class initially. But maybe it's not worth having another class just to avoid it.

Reply:
Another option would be to duplicate this in the constructor of the two subclasses.  Not ideal to duplicate code like that, but it is basically a one-liner.  I don't have a strong preference either way.

Reply:
I would duplicate it, I think. Where it is right now, it's not where I would expect to find that code if I knew I had to change it.